src/react-fetch.js

```diff
@@ -1,5 +1,10 @@
 import React from 'react/addons'
-import fetch from 'isomorphic-fetch'
+
+//this is to hack around a bug, see:
+//https://github.com/matthew-andrews/isomorphic-fetch/pull/20
+//import fetch from 'isomorphic-fetch'
+import fetch_ from 'isomorphic-fetch';
+var fetch = fetch_.bind(undefined);
```

Comment:
I think lines 2-6 can simply be replaced with:

`import 'isomorphic-fetch'`

(don't import anything into the scope, use the global fetch after importing)